velox/connectors/hive/HiveConfig.h

```diff
@@ -115,13 +115,25 @@ class HiveConfig {
   static constexpr const char* kMaxCoalescedDistanceBytes =
       "max-coalesced-distance-bytes";
 
+  /// Sets the number of prefetch rowgroups
```

Comment:
drop "Sets"; add period at the end

/// The number of rowgroups to prefetch.

ditto other places.